Sources/SwiftDocC/Semantics/Visitor/SemanticVisitor.swift

```diff
@@ -148,3 +151,11 @@ extension SemanticVisitor {
         return semantic.accept(&self)
     }
 }
+
+@available(*, deprecated)
```

Comment:
Can we add a message to all of the `@available(*, deprecated)` in this PR so we have a record of when we need to remove them?

Reply:
Added in https://github.com/swiftlang/swift-docc/pull/1039/commits/27bdcc9780107353afb4c159bbfb1969c543838a